### PR TITLE
population selection for program parameters

### PR DESCRIPTION
**Pull request by nikgraf**
https://trello.com/c/nZkYF49Q/286-1-only-list-populations-in-project-in-list-of-parameter-links

### CODE REVIEW

File: client/source/js/modules/project/create-or-edit-ctrl.js [hunk 1]
```diff
@@ -234,6 +235,31 @@ define(['./module', 'angular', 'underscore'], function (module, angular, _) {
...
+     * A program is not valid if it contains a parameter with a population not
+     * contained in the available populations
+     */
+    var findInvalidProgramParameters = function(selectedPopulations, selectedPrograms) {
+      var avialablePopulationOptions = _(selectedPopulations).pluck('short_name');
+      avialablePopulationOptions.push('ALL_POPULATIONS');
+
+      var parameterErrors = [];
+
+      _(selectedPrograms).each(function(program) {
```
Comment (Ashalynd) [inline client/source/js/modules/project/create-or-edit-ctrl.js]: Probably, this check should only be done for active parameters?

**Comment by Ashalynd** [comment]
@nikgraf please describe on the card what your changes are about, I can't see how to test them.

**Comment by nikgraf** [comment]
@Ashalynd just did so, see https://trello.com/c/nZkYF49Q/286-1-only-list-populations-in-project-in-list-of-parameter-links

**Comment by Ashalynd** [comment]
Great, it seems to work now :)

**Pull request merged**